README.md

```diff
@@ -48,10 +48,7 @@ To override configuration, invoke the `createPlugin` function prior to passing t
 ```js
 // index.js
 const createPlugin = require('@newrelic/apollo-server-plugin')
-const plugin = createPlugin({
-  captureScalars: false,
-  captureIntrospectionQueries: false
-})
+const plugin = createPlugin()
```

Comment:
might be a nit now and i did say removing stuff in here but now it might be a tad misleading to override config when there isn't an example.  it feels like we should just remove this whole area because below it says how to override config

Reply:
I see what you mean. I can remove this section and add some wording about invoking createPlugin in the Configuration section.